app/model/Forms.scala

```diff
@@ -194,7 +194,7 @@ object Forms {
     def fromSetOfWhitelistedIps(whiteListedIps: Set[String]) = Some(whiteListedIps.mkString("\n"))
 
     val form: Form[Set[String]] = Form(mapping(
-      "whitelistedIps" -> text.verifying(whitelistedIpsConstraint)
+      "allowlistedIps" -> text.verifying(whitelistedIpsConstraint)
```

Comment:
change name of val too ?

Reply:
Discussed with @AndySpaven : We're only going to change references that are returned to the browser. We're not going to fully refactor the code to use the new terminology.